configure.ac

```diff
@@ -226,13 +227,8 @@ AC_ARG_ENABLE([defaultflags],
               [enable_defaultflags=yes])
 AS_IF([test "x$enable_defaultflags" = "xyes"],
       [
-      ADD_PREPROC_FLAG([-D_DEFAULT_SOURCE])
-      ADD_PREPROC_FLAG([-D_BSD_SOURCE])
-      ADD_PREPROC_FLAG([-D_POSIX_SOURCE])
       AS_IF([test "x$enable_debug" = "xno"],
             [
-            ADD_PREPROC_FLAG([-U_FORTIFY_SOURCE])
-            ADD_PREPROC_FLAG([-D_FORTIFY_SOURCE=2])
```

Comment:
Same comment as https://github.com/tpm2-software/tpm2-abrmd/pull/632#discussion_r289333418: this is not included in `AC_USE_SYSTEM_EXTENSIONS`,  we need `AX_ADD_FORTIFY_SOURCE` instead.

Reply:
done in master already, so I rebased

Reply:
Ah yes indeed, I totally missed that...